### DISCUSSION TITLE
Водый насос

### DISCUSSION THREAD

**Discussion by Tribun-2014** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1291] [General]
There are suggestions that you can add an electromagnetic installation that disables and disables all mechanisms and equipment, the radius of damage is 50-100 blocks along the perimeter and height. I would also like this installation to be able to operate with the construction of a large antenna that are scattered throughout the world of Minecraft

**Comment by SoB1nary** [comment]
привет!!
в моде уже есть насосная станция и консоль для радаров. кстати, предложения запрещены, а автор мода не говорит по-русски.

**Reply by Tribun-2014** [reply]
На какой версий,я просто сежу на 1,7,10

**Reply by SoB1nary** [reply]
на 1.7.10, да.

**Reply by Tribun-2014** [reply]
спасибо
